src/consensus.rs

```diff
@@ -997,6 +1038,11 @@ impl RaftMessageSender {
     }
 }
 
+fn is_heartbeat(message: &RaftMessage) -> bool {
```

Comment:
Maybe we can implement this on `RaftMessage` itself.

Reply:
It's from the `raft` crate. We can do it as a trait, but that does not make much sense, IMO.

Reply:
I thought about a trait impl. But fair enough, it makes it more complicated than it's worth.